components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt

```diff
@@ -114,6 +117,13 @@ class GeckoEngine(
             get() = runtime.settings.automaticFontSizeAdjustment
             set(value) { runtime.settings.automaticFontSizeAdjustment = value }
 
+        override var automaticLanguageAdjustment: Boolean
```

Comment:
@jhugman Do we still need to initialize this field based on the default setting? To make sure the setter is called and the locale updated if the default value is `true`, similar to: https://github.com/mozilla-mobile/android-components/blob/b0824f92891b095ff50eec9234f565a41631aa9a/components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngine.kt#L159

Reply:
Good catch!

I've taken the source of truth for if the option is working in the getter by looking at the `localeUpdater` directly.